src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day_sat calendar--month-length_31">
```

Comment:
`calendar--month-length_31` - this option is not working! Check one more time scss file, there something wrong.

Reply:
fixed!